src/components/Login.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import User from './User.json';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: '',
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  handleInputChange = event => {
+    // will update state by name attribute, so state name should be same with it
+    const {value, name} = event.target;
+    this.setState({
+      [name]: value,
```

Comment:
Would you explain me this line with the [] notation? I am not sure what it does and would really like to understand it

Reply:
I put `handleInputChange` on `<input name='email'>` and `<input name='password'>`
it is just for updating state at once
`const name = event.target.name` will be email or password and value will be user's input